lib/Console/CharacterReader.php

```diff
@@ -14,17 +14,20 @@
 
 class CharacterReader
 {
+    private static bool $initialized = false;
+
     /**
      * If readline is installed, then prevent the user having to
      * press <return> in order to paginate.
      */
-    public function __construct()
+    public function initialize(): void
```

Comment:
let's make this priate an lazily initialize it when we call `read` ? this avoids "temporal coupling" and in anycase, it's not a hotpath.